vllm/lora/utils.py

```diff
@@ -30,8 +30,10 @@ def parse_fine_tuned_lora_name(name: str) -> Tuple[str, bool]:
     assert parts[0] == "base_model"
     assert parts[1] == "model"
     if parts[-1] == "weight":
-        assert parts[-2] == "lora_A" or parts[-2] == "lora_B"
-        return ".".join(parts[2:-2]), parts[-2] == "lora_A"
+        if parts[-2] == "lora_A" or parts[-2] == "lora_B":
+            return ".".join(parts[2:-2]), parts[-2] == "lora_A"
+        else:
+            return None  # Ignore Non-LoRA tensors
```

Comment:
I've been thinking if it's a good idea to have the logger warn the user that certain tensors have been skipped.